cob_default_robot_config/cob4-1/head_joint_configurations.yaml

```diff
@@ -1 +0,0 @@
-../cob4-2/head_joint_configurations.yaml
```

Comment:
I wanted to do that but then I saw that the cob4-2 urdf includes a static head: https://github.com/ipa-nhg/cob_robots/blob/indigo_dev/cob_hardware_config/cob4-2/urdf/cob4-2.urdf.xacro#L26 and the head_controllers were only for cob4-1 defined and not as softlinks https://github.com/ipa-nhg/cob_robots/tree/indigo_dev/cob_hardware_config/cob4-1/config , do you know why?